### PR TITLE
test: fix the benchmark tests

### PR DESCRIPTION
**Pull request by nazarhussain**
**Motivation**

Fix benchmark workflow. 

**Description**

Closes #issue_number

**Steps to test or reproduce**

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ on:
     branches:
       - stable
       - unstable
+  workflow_dispatch:
```
Comment (twoeths) [inline .github/workflows/benchmark.yml:20]: is this necessary?

Comment (nflaig) [reply .github/workflows/benchmark.yml:20]: I think it is just to be able to trigger it manually, might be useful to have

**Review by nflaig — APPROVED** [review]

File: .github/workflows/benchmark.yml (hunk 1 above)
Comment (nazarhussain) [reply .github/workflows/benchmark.yml:20]: In some discussions @dapplion suggested to open draft PRs to test some idea. For that purpose it's required sometime to run some workflows manually on branches.

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.13.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.13.0) :tada:

**Pull request merged**